src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days >= 7) {
+    result = days * 40 - 50;
+
+    return result;
+  } else if (days >= 3) {
```

Comment:
```suggestion
  }

  if (days >= 3) {
```
when you return inside an `if` block you don't need `else` - it increases readability